.github/workflows/cleanup-stale-performance-data.yml

```diff
@@ -1,44 +1,38 @@
-name: Cleanup stale performance test data
-
+name: Cleanup Stale Performance Test Data 🧹️
 on:
   schedule:
-    - cron: '0 4 * * 1'  # Run every Monday at 4:00 AM UTC
```

Comment:
ide can do this as well:

<img width="366" height="47" alt="Image" src="https://github.com/user-attachments/assets/3f974a51-25d0-48ac-bf0d-406869e8eebd" />